### DISCUSSION TITLE
Version 3.* feature requests

### DISCUSSION THREAD

**Discussion by dwainscheeren** [dwainscheeren/dwains-lovelace-dashboard#456] [Ideas]
If you got a good idea for a next 3.* release please post it in this discussion :)
 
Currently I got the following things in mind:
- Sort the favorites on the homepage
- Sort custom cards in the areas
- Sort custom cards in the devices view
- Sort more pages
- Add missing icons for some devices
- Make the homepage house information badges editable and add functionality to add own badges (templated)
- Add custom badges to the area button (templated)

**Comment by thebatfink** [comment]
- Double click the room icon to turn on lights again like v2.0 and not have to hit the tiny light bulb icon
- The ability to toggle off the action of clicking the plug / switch icon in the room icons triggering a service turn off on all the switches. I do use the light bulbs icon to turn on the lights (as double click doesn't seem to work) but I keep catching the plug and turning stuff off accidentally!

**Reply by dwainscheeren** [reply]
Double click the room icon to turn on lights again like v2.0 and not have to hit the tiny light bulb icon
is done for 3.1

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by danalex0829** [comment]
- add a shortcut button to restart home assistant and a shortcut button to access the ha config dashboard from the settings (gear icon on top-right) as to not have to go through the home assistant sidebar options

**Reply by dwainscheeren** [reply]
Not sure why I would make this You can really acces settings in the sidemenu or make a own shortcut in dd with a custom card to restart ha?

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by brandon-claps** [comment]
Would be awesome to have a setting on an entity whether you want it to reflect on the main area icon for its state. For example, I have a subwoofer that is on all the time and I wouldn't want it's on state showing on the area card, just the things I care about seeing if they've been left on.

In addition, allow an entity to not show in the count at the top of the dashboard icons. For example, if I had a specific lock that was always unlocked, I wouldn't want it to show as unlocked in the dashboard.

This is all in the spirit of focusing on only those things that are important.

**Reply by thebatfink** [reply]
This would also help my issue where I have always on switches like server etc. (so I can monitor power usage) that I’ve accidentally turned off with the area icon. I like your idea better.

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by bacco007** [comment]
* Can the tiles on the More Page be made smaller on larger displays?
* Can support for [Apexcharts Templates](https://github.com/RomRider/apexcharts-card#configuration-templates) be added (a bit like it has for button-card)?

**Reply by dwainscheeren** [reply]
Added both thins in 3.1 :)
Its `apexcharts_card_templates`

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by Adrenocortico** [comment]
- Sort the people in the homepage.
- Edit the popup view of person.

**Reply by dwainscheeren** [reply]
Edit the popup view of person.
This is already possible by adding the persons to an area and go to devices -> persons and then edit the popup there. Afterwards you can also remove them from the area if you prefer but keep the custom popup.

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by dimmuboy** [comment]
- Bring back haptic feedback

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by xBourner** [comment]
Changing column width on tablets. The area space should be smaller. 2 columns instead of 3. so the content part can have 4 instead of 3.

area icons take to much space in landscape mode right now.

**Comment by Peenpower** [comment]
Able to delete or disable weather on the main screen

**Reply by brandon-claps** [reply]
That can be done already

**Reply by Peenpower** [reply]
Just created also an issue with an movie 
https://youtu.be/HcANY6wYr-g

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by brandon-claps** [comment]
When using the icons at the top of the main dashboard, it reflects the DD name. It would be good if you can customize the name separate for this and the devices screen. 

Use case: when you’re in an area you likely want to just call it “TV”, and there are more than one “TV” in different areas. When using the icon at the top or the entity screen, you can’t identify which TV it is
Specifically. 

An alternative option would be adding the area to the card in those views to identify.

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by rotary65** [comment]
In order to save people with large HA systems significant time, please consider adding an advanced option when adding the integration that would enable the users to  set preferences for the initial auto configuration. The preferences could include enable/hide/disable by entity type. DD would then build the configuration automatically based on the preferences.

This would give the user some control over the setup work

**Reply by romant17** [reply]
This would be very helpful. Right now, a large number of sensors e.g .identify,.lqi,.rssi, .power_meter, .energy_meter show up that I would want to exclude.

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by Vansmak** [comment]
It would be nice to be able to link an Areas Icon, the large one, to an entity and allow it to change based on state of the entity.  For example I have a custom sensor for my front door and its lock that on custom button card shows one of 3 states\icons. open, closed unlocked, closed locked.

**Comment by xBourner** [comment]
please add binary sensors with device class "running" to the header. the icon should be the same of the entity in HA.
this would be a nice option for some custom devices like washing machines.

**Comment by xBourner** [comment]
please change layout in blueprint editor that have checkboxes. currently the checkbox is on the right side of the description and also its not on the same height like the description.

it would look better when both are on the same height and the checkbox will be on the left side and next to it the description

**Comment by xBourner** [comment]
please add some sorting of blueprints. now that all blueprint types are shown in the same place it would be better for visibility when you could add sorting by blueprint type. within the types it should be sorting alphabetically.

so on more page there should be more page blueprints on top which are sorted alphabetically. then comes cards and then replace cards.
on entity cards there should be card blueprints (sorted alphabetically) then replace cards and on bottom should be more page blueprints.

the blueprints you can use in the section should be always on top and then sorted alphabetically. so users with many blueprints dont need to search that long

**Reply by dwainscheeren** [reply]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by dwainscheeren** [comment]
I've lost a little bit track of all feature request here in the discussion combined with people opening them as an issue. Can you all please open an Github Issue with [FR] your feature request as name :) Many thanks!!!

**Comment by bobhogs** [comment]
Is it possible to rename the groups on the device page? For instance if we did not want the Alarm Control Panel could we rename it something else more useful to our individual usecase

**Comment by kekec777** [comment]
Make added entities movable ?
<img width="554" alt="DwainsNotMoveable" src="https://user-images.githubusercontent.com/20354802/187430794-9c3cce06-6933-46f1-828a-37ecaca6772f.png">

**Comment by KevLille** [comment]
Hi Dwain

Very impressive work, I love it.

Some suggestions for new release:
- be able to change the order/name/icon of the top bar (including the by default pages)
- be able to change the size/color/icon/background etc.. on the area cards
- be able to chose the order of the floors (currently ordered by alphabetic sorting)
-  be able to use the layout-card

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
